tests/functional/api/groups/test_create.py

```diff
@@ -9,145 +9,101 @@
 
 class TestCreateGroup:
     def test_it_returns_http_200_with_valid_payload(self, app, token_auth_header):
-        group = {"name": "My Group"}
-        res = app.post_json("/api/groups", group, headers=token_auth_header)
+        res = app.post_json(
+            "/api/groups", {"name": "My Group"}, headers=token_auth_header
+        )
 
         assert res.status_code == 200
 
     def test_it_ignores_non_whitelisted_fields_in_payload(self, app, token_auth_header):
-        group = {"name": "My Group", "organization": "foobar", "joinable_by": "whoever"}
-        res = app.post_json("/api/groups", group, headers=token_auth_header)
+        res = app.post_json(
+            "/api/groups",
+            {"name": "My Group", "organization": "foobar", "joinable_by": "whoever"},
+            headers=token_auth_header,
+        )
 
         assert res.status_code == 200
 
     def test_it_returns_http_400_with_invalid_payload(self, app, token_auth_header):
-        group = {}
```

Comment:
The tests repeatedly refer to the payload / body as `group` which made refactoring difficult as this is also the name of the `group` fixture. These were in-lined or renamed where the variable is actually re-used.